sklearn/model_selection/_validation.py

```diff
@@ -281,7 +281,13 @@ def cross_val_score(estimator, X, y=None, groups=None, scoring=None, cv='warn',
     scoring : string, callable or None, optional, default: None
         A string (see model evaluation documentation) or
         a scorer callable object / function with signature
-        ``scorer(estimator, X, y)``.
+        ``scorer(estimator, X, y)`` which should return only
+        a single value.
+
+        Similar to :func:`cross_validate`
+        but only a single metric is permitted.
+
+        If None, the estimator's default scorer (if available) is used.
```

Comment:
Just say "the estimator's score method"

Reply:
@jnothman I thought this was more consistent with `cross_validate` documentation [here](https://github.com/scikit-learn/scikit-learn/blob/19c8af6d6a9528162fd41d8a3363b807e337d63d/sklearn/model_selection/_validation.py#L74) but if you still want me to change, then I will

Reply:
I'm happy with it being consistent... I'd be more happy with them all being changed :) maybe in a subsequent PR?

Reply:
@jnothman a separate PR is best as it hits files other than `_validation.py` as well, such as `scorer.py` in sklearn/metrics/ and `_search.py` in the same folder. I can get started on that PR if you can merge this one as it may conflict with this file.